### DISCUSSION TITLE
Showing on laptop, 404 error on mobile

### DISCUSSION THREAD

**Discussion by nanderspeerstra** [community/community#22562] [Other Feature Feedback, Questions, & Ideas]
<p>My website, <a href="http://nanderspeerstra.github.io" rel="nofollow noopener">nanderspeerstra.github.io</a>, is showing as expected on my laptop: Chrome, Firefox, Edge; all working fine. However, when I try to visit my website via mobile (Android, iOS), I get the ‘404 There isn’t a GitHub Pages site here.’ error. Is there some setting that I did not set correctly?</p>

**Comment by nadiajoyce** [comment]
<p><a class="mention" href="/nanderspeerstra">@nanderspeerstra</a> Your site appears to be pulling up just fine for me both in Chrome and Safari on both my desktop computer and mobile. Are you still experiencing this issue or did you find a solution since you posted this?</p>

**Comment by nanderspeerstra** [comment, marked as answer]
<p><a class="mention" href="/nadiajoyce">@nadiajoyce</a>, if you were able to see my website correctly, I figured that it had to be my iPhone… So I removed Safari’s cache/cookies regarding my website and now it’s working fine again. I should’ve thought of that step earlier. Thanks for responding :) </p>
<p>Still, interesting to see that changing the index.html page a bit (only HTML changes) and then keeping cookies/cache triggers a 404 error. But not interesting enough to keep this thread open. Case closed, thanks again!</p>

**Comment by jw1207j** [comment]
<p>I have now the same problem that I cannot see the the page execept “404” when I tried to open on the mobile.</p>
<p>It would be greatful for you to solve the problem in details. </p>

**Comment by nadiajoyce** [comment]
<p>I would first recommend checking our <a href="https://help.github.com/articles/user-organization-and-project-pages/#user-and-organization-pages-sites" rel="nofollow noopener">help articles on how to setup GitHub Pages</a>. Please check to make sure you are using the correct branch and also applying the correct repo name to make your Pages site work (e.g. <a href="http://jw1207j.github.io" rel="nofollow noopener">jw1207j.github.io</a> should be your repo name for your Pages site). If you’re still having difficulty, I would recommend reaching out to <a href="https://github.com/contact" rel="nofollow noopener">private support</a>, as they will be able to look more deeply into your specific account to see what may be going on.</p>
